Readme.md

```diff
@@ -0,0 +1,222 @@
+
+## Rendering
+### Sprites
+The main way to render images in Pixelot is to use the provided `Sprite` component, which takes as argument the path to the image file (which must be contained within the project folder). **The sprites will be drawn true to the size of the original image.** Sprites can also be rotated and have their rotation anchor point set.
+
+### Layers
+Rendering layers control the order in which sprites will be drawn. Layers are drawn in order from top to bottom, which means lower layers will appear over higher layers. New sprite layers can be added by click 'Add Layer' in the left sidebar and selecting the 'Sprite Layer'. When creating a sprite component, specify the name of the layer that the sprite should be drawn on. By default, newly added entities will be added to the 'No Layer' tab.  
+Within the layers, the render order can be controlled through the sprite z-index, which is set through the `z_index` property of the sprite component. The default z-index is 0, and sprites with a higher z-index will be drawn on top of sprites with a lower z-index. 
+
+### Text
+To display text use the `Text` component, which takes as argument the text to be displayed. Currently there is no support for using custom fonts, but the engine has some built in fonts. Which font to use can also be specified as an argument to the `Text` component.  
+**The fonts available are:**
+- default
+  
+### Tilemaps
+Pixelot supports rendering tilemap layers. Tilemaps should be created in 'Tiled' (https://www.mapeditor.org/). In order to export in the correct format, ensure that the tilesets are embedded in the `json` file (can be toggled when opening a tileset in Tiled). Additionally, make sure to export it directly into the projects directory, as Tiled will configure the paths to the tilesets relative to the `json` file. As such, also make sure that the tileset images are being stored within the projects directory. **This means that if you want to move the location of the tilemap `.json`, or the tileset images, you should delete it and re-export from Tiled to ensure the relative paths stay valid**
+
+Pixelot supports arbitrary layers and tilesets, but won't support any custom properties, or transparency colours set in Tiled. In order to add transparency, make tilesets with transparent backgrounds in an image editor, and then export them as `.png` files.
+
+To retrieve objects from the object layer, use the `getObject` function of the `TileMapObjectLayer` class. 
+To retrieve an object layer from the tilemap, use the `getObjectLayer` function of the `TileMapLayer` class.
+
+
+### Shaders
```

Comment:
Empty subheading?

Reply:
Yeah I left this in for when we eventually would add the ability to apply shaders through the UI.